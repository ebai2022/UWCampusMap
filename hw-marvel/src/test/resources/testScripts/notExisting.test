# Testing a node that does not exist in the graph

LoadGraph g strangers.csv

# char1 doesn't exist
FindPath g Imposter Fake

# char2 doesn't exist
FindPath g Ethan Edward

# both chars don't exist
FindPath g Edward Ralen