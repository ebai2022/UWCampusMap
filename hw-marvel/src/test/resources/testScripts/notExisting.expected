# Testing a node that does not exist in the graph

loaded graph g

# char1 doesn't exist
unknown: Imposter

# char2 doesn't exist
unknown: Edward

# both chars don't exist
unknown: Edward
unknown: Ralen